evap/__main__.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+from django.conf import settings
+from django.core.management import execute_from_command_line
+
+
+def main():
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "evap.settings")
```

Comment:
I think the environment variable must be set before we import `settings`.

Reply:
I believe that it's configured on first access to an attribute:
```python
def main():
    assert not settings.configured
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "evap.settings")
    settings.DATADIR.mkdir(exist_ok=True)
    assert not settings.configured
    execute_from_command_line(sys.argv)
    assert not settings.configured
```
with `./manage.py help` fails on the second assertion. We could add the first assertion, or we could also move the import back in here. I think that it's fine at the top, since we can't guarantee that the settings haven't been configured from just this file anyways